src/Bedrock.Framework.Experimental/Protocols/Memcached/Serializers/ISerializer.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.Serialization.Formatters.Binary;
+using System.Text;
+
+namespace Bedrock.Framework.Experimental.Protocols.Memcached.Serializers
+{
+    public interface ISerializer<T>
+    {
+        byte[] Serialize(T src);
+        T Deserialize(ReadOnlyMemory<byte> src);
+    }
+    public class StringSerializer : ISerializer<string>
+    {
+
+        public string Deserialize(ReadOnlyMemory<byte> src) => Encoding.UTF8.GetString(src.Span);
+        public byte[] Serialize(string src) => Encoding.UTF8.GetBytes(src);
+    }
+    public class BinarySerializer<T> : ISerializer<T>
+    {
+        public T Deserialize(ReadOnlyMemory<byte> src)
+        {
+            using var ms = new MemoryStream(src.ToArray());
+            var formatter = new BinaryFormatter();
```

Comment:
Besides @davidfowl's comment above, FYI https://github.com/dotnet/runtime/issues/29976

Reply:
Ok i miss that.  Anyway, i removed serializer feature, to follow @davidfowl's comment